app/layout.tsx

```diff
@@ -32,11 +44,11 @@ export default async function RootLayout({
           {children}
           <footer className="flex items-center justify-between">
             <div className="flex gap-2 items-center">
-              {installer?.metadata?.copyright_markdown ? (
+              {metadata.copyright_markdown ? (
                 <div
                   dangerouslySetInnerHTML={{
                     __html: markdown.makeHtml(
-                      installer?.metadata?.copyright_markdown,
+                      metadata.copyright_markdown,
```

Comment:
I'm not entirely convinced we need to support markdown here (or anywhere else, for that matter.) It's neat, but if we want to use the API for other client experiences, it may not be flexible enough.